deploy/chart/values.yaml

```diff
@@ -83,6 +83,9 @@ tolerations:
 # Enable Linode token health checker
 # tokenHealthChecker: true
 
+# Default NodeBalancer type to create (default is "common")
```

Comment:
might also want to note premium is the other option here

Reply:
Updated.